pkg/antctl/command_definition_test.go

```diff
@@ -108,13 +108,13 @@ func TestCommandDefinitionGenerateExample(t *testing.T) {
 	for k, tc := range map[string]struct {
 		use          string
 		cmdChain     string
-		singleObject bool
+		outputObject OutputType
```

Comment:
why not use outputType as variable name?

Reply:
Updated.